### PR TITLE
OCPBUGS-32233: Not able to enable repositories during entitled build in OCP Cluster on IBM-Z

### PR DESCRIPTION
**Pull request by joselsegura**
## Categories
- [X] Bugfix
- [ ] Data Enhancement
- [ ] Feature
- [ ] Backporting
- [ ] Others (CI, Infrastructure, Documentation)

## Sample Archive
N/A

## Documentation
N/A

## Unit Tests
N/A

## Privacy
Yes. There is no new data collected.

## Breaking Changes
No

## References
https://issues.redhat.com/browse/OCPBUGS-32233

### CODE REVIEW

File: pkg/ocm/sca/sca.go [hunk 1]
```diff
@@ -185,41 +185,56 @@ func (c *Controller) createSecret(ocmData *Response) (*v1.Secret, error) {
...
 	}
 	return s, nil
 }
 
+// getArch check the value of GOARCH and return a valid representation for
+// OCM certificates API
+func getArch() string {
+	validArchs := map[string]string{
+		"amd64": "x86_64",
+		"i386":  "x86",
```
Comment (tremes) [inline pkg/ocm/sca/sca.go:213]: So the remaining archs don't require additional mapping right?

Comment (joselsegura) [reply pkg/ocm/sca/sca.go:213]: Yes, I tried the endpoint with other supported architectures and they don't need any translation mapping like these two.

**Comment by tremes** [comment]
Thank you! 
/approve
/lgtm

**Comment by tremes** [comment]
Some missing files in the archive. It's not related to this PR. 
/override ci/prow/insights-operator-e2e-tests

**Pull request merged**